extensions/tiled-assets/README.md

```diff
@@ -0,0 +1,138 @@
+| max_tile_rol  | number  | Maximum tile column index valid for this layer. If not specified it uses the the one from the referenced tile matrix. |
+
+### Pixel Buffer Object
+
+Pixel buffer objects allow the definition of image boundarys, so that the internal tiles may overlap. When using this information, the clients may be able to reduce the number of requests.
+
+| Field Name    | Type    | Description                                                                                                        |
+| ------------- | ------- | ------------------------------------------------------------------------------------------------------------------ |
+| top           | number  | The size of the pixel-buffer in the top border of the image. Default is `0`.                                       |
+| left          | number  | The size of the pixel-buffer in the left border of the image. Default is `0`.                                      |
+| bottom        | number  | The size of the pixel-buffer in the bottom border of the image. Default is `0`.                                    |
+| right         | number  | The size of the pixel-buffer in the right border of the image. Default is `0`.                                     |
+| border_top    | boolean | Whether or not the pixel-buffer is included images on the top border of the first tile row. Default is `true`.     |
+| border_left   | boolean | Whether or not the pixel-buffer is included images on the left border of the first tile column. Default is `true`. |
+| border_bottom | boolean | Whether or not the pixel-buffer is included images on the bottom border of the last tile row. Default is `true`.   |
+| border_right  | boolean | Whether or not the pixel-buffer is included images on the right border of the last tile column. Default is `true`. |
+
+## Item fields
+
+| Field Name          | Type                                                         | Description                                                        |
+| ------------------- | ------------------------------------------------------------ | ------------------------------------------------------------------ |
+| asset_templates     | Map<string, [Asset Template Object](#asset-template-object)> | An array of Asset Template objects that denote templated arguments |
+
+### Asset Template Object
+
+An asset template is an object that contains a link template to data associated with the Item that can be downloaded or streamed. It is allowed to add additional fields.
+
+| Field Name    | Type   | Description                                                                           |
+| ------------- | ------ | ------------------------------------------------------------------------------------- |
+| href_template | string | **REQUIRED.** Link template to the asset object. Relative and absolute links are both allowed. All parts of the template within curly braces `{}` are meant to be template substitution parameters to be replaced |
```

Comment:
Is this compatible with https://tools.ietf.org/html/rfc6570? Should it be mentioned?

Reply:
I was not aware of this RFC. It actually looks very promising, and we should definitely find a way to incorporate it.

The only issue I see here is with the use of "namespaced" template parameters, such as `eo:bands` which is not permitted with the RFC (there the colon separates the variable from its `max-length` field). Do you have a suggestion for that issue?

Reply:
I haven't looked into the RFC yet, but it seemed relevant so linked it here. If there are big burdens of adoption, we may skip it.

Reply:
I read that spec in detail now. I think it is very interesting, but might not be all too useful for our purposes. The spec states that there are other mechanisms of templating, specifically mentioning OpenSearch and WSDL. 

Also the WMTS spec defines a simple templating mechanism, which this extension actually builds upon. So I'd rather reference that, if it is okay with you. It can be found here: http://docs.opengeospatial.org/is/13-082r2/13-082r2.html#17